omod/src/test/java/org/openmrs/module/bedmanagement/rest/resource/AdmissionLocationResourceTest.java

```diff
@@ -132,8 +132,8 @@ public void shouldReturnAdmissionLocationByUuidWithFullRepresentation() throws E
 		SimpleObject location = deserialize(handle(request));
 		
 		Assert.assertEquals("19e023e8-20ee-4237-ade6-9e68f897b7a9", PropertyUtils.getProperty(location.get("ward"), "uuid"));
-		Assert.assertEquals(6, location.get("totalBeds"));
-		Assert.assertEquals(2, location.get("occupiedBeds"));
+		Assert.assertEquals(Integer.valueOf(6), location.get("totalBeds"));
+		Assert.assertEquals(Integer.valueOf(2), location.get("occupiedBeds"));
```

Comment:
All of these were giving compilation errors due to ambiguous method calls.  Likely due to change from Java 7 -> 8.